spec/lib/tasks/local_census_records_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require Rails.root.join("db", "migrate", "20190530082138_add_unique_index_to_local_census_records")
+
+describe "LocalCensusRecord tasks" do
+  let(:run_rake_task) do
+    Rake::Task["local_census_records:remove_duplicates"].reenable
+    Rake.application.invoke_task("local_census_records:remove_duplicates")
+  end
+
+  describe "#remove_duplicates" do
+    around do |example|
+      ActiveRecord::Migration.suppress_messages do
+        example.run
+      end
+    end
+
+    before { AddUniqueIndexToLocalCensusRecords.new.down }
+    after { AddUniqueIndexToLocalCensusRecords.new.up }
+
+    it "Remove all duplicates keeping older records" do
+      record1 = create(:local_census_record, document_type: "1", document_number: "#DOCUMENT_NUMBER")
+      record2 = create(:local_census_record, document_type: "2", document_number: "#DOCUMENT_NUMBER")
+      duplicated_record1 = build(:local_census_record, document_type: "1",
+                                                      document_number: "#DOCUMENT_NUMBER")
```

Comment:
Is the intention to align the `d` from `document_number` with the `d` from `document_type`? :thinking: Just curious; we don't have consistent criteria about multi-line alignments.

Reply:
Yes it was, but refactoring i miss one whitespace 😅 ! Personally prefer
```
duplicated_record1 = build(:local_census_record, document_type: "1", 
                                                 document_number: "#DOCUMENT_NUMBER")
``` 

over

```
duplicated_record1 = build(:local_census_record, document_type: "1", 
  document_number: "#DOCUMENT_NUMBER")
```

Reply:
Here finally replaced `duplicated_record1` with `dup_record1` so there is no need to use multiline and at the same time accomplish Rubocop max line length rule.